zerver/lib/actions.py

```diff
@@ -2980,9 +2980,34 @@ def do_remove_streams_from_default_stream_group(realm: Realm, group_name: Text,
     group.save()
     notify_default_stream_groups(realm)
 
+def do_change_default_stream_group_name(realm: Realm, group_name: Text, new_group_name: Text) -> None:
```

Comment:
Some of your `def` line names get quite long; it'd be best to wrap them.  We're trying to work towards nothing longer than 100 characters in the Python codebase, and things only hitting that when it makes the code more readable.